src/components/StoreRevokeBlock.js

```diff
@@ -1,80 +1,162 @@
 import React, { Component } from "react";
+/** @jsx jsx */
```

Comment:
Maybe we should change the color for the revoke button to indicate that's it's a non reversible action and user hould be cautious on it

=> idea is to remove extra layers of checking (modal, automatic revoke when D&D)

For instance can use https://getbootstrap.com/docs/4.0/components/alerts/

Reply:
Added a red button to show that it is an irreversible action.